.vscode/settings.json

```diff
@@ -60,5 +60,8 @@
         "text",
         "yaml",
         "yml"
-    ]
+    ],
+    "[csharp]": { 
```

Comment:
Let me know if this ok to add in. I ussually have this always set in VSCode when making changes to csharp files.